docs/reference-architectures/virtual-machines-linux/multi-vm.md

```diff
@@ -3,104 +3,93 @@ title: Run load-balanced VMs on Azure for scalability and availability
-* **Network interfaces (NICs)**. Each VM has a NIC to connect to the network.
+* **Load balancer rules**. Used to distribute network traffic among all the VMs in the back-end address pool.
+* **VM Scaleset**. A [VM scaleset][vm-scaleset] contains a set of VMs with identical settings, used to host the same workload. Scaelsets allow to scale the number of VMs out and in manually, or automatically based on predefined rules.
+* **Availability set**. The [availability set][availability set] contains the VMs. This makes the VMs eligible for the [availability service level agreement (SLA) for Azure VMs][vm-sla]. For the SLA to apply, you need a minimum of two VMs in the same availability set.
 * **Storage**. If you are not using [managed disks](/azure/storage/storage-managed-disks-overview), storage accounts hold the VM images and other file-related resources, such as VM diagnostic data captured by Azure.
 
 You can download a [Visio file](https://aka.ms/arch-diagrams) of this architecture.
 
 > [!NOTE]
 > Azure has two different deployment models: [Resource Manager][resource-manager-overview] and classic. This article uses Resource Manager, which Microsoft recommends for new deployments.
->
 
 ## Recommendations
 
 Your requirements might differ from the architecture described here. Use these recommendations as a starting point. 
 
-
 ### Availability set recommendations
 
 Create at least two VMs in the availability set, to support the [availability SLA for Azure VMs][vm-sla]. The Azure load balancer also requires that load-balanced VMs belong to the same availability set.
 
-Each Azure subscription has default limits in place, including a maximum number of VMs per region. You can increase the limit by filing a support request. For more information, see [Azure subscription and service limits, quotas, and constraints][subscription-limits].  
+Each Azure subscription has default limits in place, including a maximum number of VMs per region. You can increase the limit by filing a support request. For more information, see [Azure subscription and service limits, quotas, and constraints][subscription-limits].
 
 ### Network recommendations
 
 Place the VMs within the same subnet. Do not expose the VMs directly to the Internet, but instead give each VM a private IP address. Clients connect using the public IP address of the load balancer.
 
+If you need to remote into the VMs behind the load balancer, you should consider adding a single VM as a bastion host/jumpbox with a public IP address you can remote into. And then remote into the VMs behind the load balancer from the jumpbox. Alternatevily, you configure inbound NAT rules in the load balancer for the same prupose. However, having a jumpbox is a better solution when you are hosting n-tier workloads, or multiple workloads.
```

Comment:
"remote into" -> log into

"you should consider" -> consider

"you configure" -> you can configure